### ISSUE TITLE
IllegalStateException: The Processor for the element type END_EVENT is already migrated 

### ISSUE THREAD

**Issue by pihme** [camunda-cloud/zeebe#7842]
**Describe the bug**
Observed in logs: https://console.cloud.google.com/errors/CKCR7Y-Xzqq6Eg?service=zeebe&time=P7D&refresh=off&project=camunda-cloud-240911

**Log/Stacktrace**
<details><summary>Full Stacktrace</summary>
 <p>

```
java.lang.IllegalStateException: The Processor for the element type END_EVENT is already migrated no need to call #transitionToActivating again this is already done in the BpmnStreamProcessor for you. Happy to help :) 
	at io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnStateTransitionBehavior.verifyIncidentResolving(BpmnStateTransitionBehavior.java:143) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.bpmn.behavior.BpmnStateTransitionBehavior.transitionToActivating(BpmnStateTransitionBehavior.java:76) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.bpmn.BpmnStreamProcessor.processEvent(BpmnStreamProcessor.java:142) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.bpmn.BpmnStreamProcessor.lambda$processRecord$1(BpmnStreamProcessor.java:128) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.Either$Right.ifRightOrLeft(Either.java:344) ~[zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.bpmn.BpmnStreamProcessor.processRecord(BpmnStreamProcessor.java:125) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.TypedRecordProcessor.processRecord(TypedRecordProcessor.java:54) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.lambda$processInTransaction$2(ProcessingStateMachine.java:258) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.db.impl.rocksdb.transaction.ZeebeTransaction.run(ZeebeTransaction.java:84) ~[zeebe-db-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processInTransaction(ProcessingStateMachine.java:248) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processEvent(ProcessingStateMachine.java:216) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.tryToReadNextEvent(ProcessingStateMachine.java:191) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.readNextEvent(ProcessingStateMachine.java:183) ~[zeebe-workflow-engine-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorJob.invoke(ActorJob.java:73) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorTask.execute(ActorTask.java:122) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:94) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorThread.doWork(ActorThread.java:78) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
	at io.camunda.zeebe.util.sched.ActorThread.run(ActorThread.java:191) [zeebe-util-1.2.0-alpha2.jar:1.2.0-alpha2]
```

</p>
</details>

**Environment:**
 - environment: Camunda Cloud, 1.2.0-alpha2

**Comment by npepinpe** [comment]
Please extract the model so we can figure out how likely it is that users will run into this.

**Comment by pihme** [comment]
Process model: https://github.com/camunda-cloud/operate/blob/77f37e2413d8a3ac0faeb3381caaa5b5f35574c9/data-generator/src/main/resources/develop/timerProcess_v_1.bpmn

**Comment by Zelldon** [comment]
![diagram_1](https://user-images.githubusercontent.com/2758593/133748747-f1c2897e-6873-4695-8522-4ff3e51ae033.png)

**Comment by Zelldon** [comment]
looks like a simple and "common" model :sweat_smile: @npepinpe

**Comment by npepinpe** [comment]
Yeah, I'm surprised this doesn't happen more often :exploding_head:

**Comment by Zelldon** [comment]
@pihme could you find out whether the cluster was upgraded during execution?

**Comment by Zelldon** [comment]
Maybe it was created with an earlier version, deployed and since the process is recurring it might happen it started already before and then the cluster was upgraded and then completed? Maybe this cause something like this?!

**Comment by pihme** [comment]
What I do not understand is why it only occurred twice. It should be occurring every 8 minutes.

**Comment by Zelldon** [comment]
Not really. It only affects instances which have been created before or inbetween.

**Comment by Zelldon** [comment]
But can also be related to building state on followers, previously we had not batch replay which can cause inconsistent state.

**Comment by pihme** [comment]
According to Ralf the model was deployed this Tuesday 14th of September. (the error occurred on 16th of September)

The cluster has NOT been updated since then.

**Comment by saig0** [comment]
If the error happens shortly before the node switched from follower to leader then it could indicate that it is related to the fact that the follower didn't read in batches yet (#7706).

**Comment by Zelldon** [comment]
@pihme could you check the metrics whether it was restarted or a leader change happened before the error occurred,

**Comment by pihme** [comment]
Occurred on broker 2/partition 2.

Timeline of events (that broker and partition):
* 16:22 Broker restart
* 16:40 transition to LEADER
* 16:49 Error occurs

Log
```
I 2021-09-16T14:40:31.224043Z RaftServer{raft-partition-partition-2} - Transitioning to LEADER 
I 2021-09-16T14:40:31.230905Z RaftServer{raft-partition-partition-2} - Found leader 2 
I 2021-09-16T14:40:31.297302Z Close appender for log stream logstream-raft-partition-partition-2 
I 2021-09-16T14:40:31.298627Z On closing logstream logstream-raft-partition-partition-2 close 1 readers 
I 2021-09-16T14:40:31.484688Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@43075e81)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@3819293f, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@37ef86e2, configuration: Configuration(false)] 
I 2021-09-16T14:40:31.487252Z Recovered state of partition 2 from snapshot at position 33770 
I 2021-09-16T14:40:31.487978Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@61b17016)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@183bc9f7, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@699a5b1e, configuration: Configuration(false)] 
I 2021-09-16T14:40:31.489746Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@22891859)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@48d2fdde, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@ee0e142, configuration: Configuration(false)] 
I 2021-09-16T14:40:31.491170Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@3519192b)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@5fb88401, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@dbae25, configuration: Configuration(false)] 
I 2021-09-16T14:40:31.493638Z Processor starts replay of events. [snapshot-position: 33770, replay-mode: PROCESSING] 
I 2021-09-16T14:40:31.495165Z Processor finished replay at event position 33773 
I 2021-09-16T14:40:31.497989Z Starting processing of migration tasks (use LogLevel.DEBUG for more details) ...  
I 2021-09-16T14:40:31.512094Z Skipping ProcessMessageSubscriptionSentTimeMigration migration (1/2).  It was determined it does not need to run right now. 
I 2021-09-16T14:40:31.512848Z Skipping MessageSubscriptionSentTimeMigration migration (2/2).  It was determined it does not need to run right now. 
I 2021-09-16T14:40:31.513203Z Completed processing of migration tasks (use LogLevel.DEBUG for more details) ...  
I 2021-09-16T14:40:31.753655Z Exporter opened 
W 2021-09-16T14:40:32.213653Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13006, prevLogTerm=9, entries=0, commitIndex=13006} to 0 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:40:32.235875Z RaftServer{raft-partition-partition-2} - ConfigureRequest{term=10, leader=2, index=0, timestamp=1631623859890, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}]} to 0 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:40:33.237900Z RaftServer{raft-partition-partition-2} - ConfigureRequest{term=10, leader=2, index=0, timestamp=1631623859890, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}]} to 0 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:40:48.486889Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13006, prevLogTerm=9, entries=0, commitIndex=13007} to 0 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Expected to send a message with subject 'raft-partition-partition-2-append' to member '0', but member is not known. Known members are '[Member{id=1, address=zeebe-1.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwABAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMS56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAAIAAAABDAABAQAAAAUAAAAAAAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}, Member{id=2, address=zeebe-2.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwACAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMi56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAQIAAAAADAABAgAAAAoAAAAAAAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}]'. 
I 2021-09-16T14:40:56.085952020Z {} 
W 2021-09-16T14:40:59.486848Z RaftServer{raft-partition-partition-2} - ConfigureRequest{term=10, leader=2, index=0, timestamp=1631623859890, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}]} to 0 failed: java.util.concurrent.CompletionException: java.lang.IllegalStateException: Failed to bootstrap client (address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 cannot be resolved) 
W 2021-09-16T14:40:59.487892Z RaftServer{raft-partition-partition-2} - ConfigureRequest{term=10, leader=2, index=0, timestamp=1631623859890, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}]} to 0 failed: java.util.concurrent.CompletionException: java.lang.IllegalStateException: Failed to bootstrap client (address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 cannot be resolved) 
W 2021-09-16T14:40:59.736772Z RaftServer{raft-partition-partition-2} - ConfigureRequest{term=10, leader=2, index=0, timestamp=1631623859890, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.890Z}]} to 0 failed: java.util.concurrent.CompletionException: java.lang.IllegalStateException: Failed to bootstrap client (address zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 cannot be resolved) 
I 2021-09-16T14:41:01.129109461Z {} 
I 2021-09-16T14:42:06.778969Z Finished taking temporary snapshot, need to wait until last written event position 73415 is committed, current commit position is 0. After that snapshot will be committed. 
I 2021-09-16T14:42:06.800812Z Committed new snapshot 25162-5-73412-73415 
I 2021-09-16T14:47:06.788790Z Finished taking temporary snapshot, need to wait until last written event position 73464 is committed, current commit position is 0. After that snapshot will be committed. 
I 2021-09-16T14:47:06.815333Z Committed new snapshot 25178-5-73462-73464 
W 2021-09-16T14:47:22.742211Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13007, prevLogTerm=10, entries=0, commitIndex=13007} to 1 failed: java.util.concurrent.CompletionException: io.atomix.cluster.messaging.MessagingException$NoRemoteHandler: No remote message handler registered for this message 
W 2021-09-16T14:47:22.987247Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13007, prevLogTerm=10, entries=0, commitIndex=13007} to 1 failed: java.util.concurrent.CompletionException: io.atomix.cluster.messaging.MessagingException$NoRemoteHandler: No remote message handler registered for this message 
W 2021-09-16T14:47:22.988469Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13007, prevLogTerm=10, entries=0, commitIndex=13007} to 1 failed: java.util.concurrent.CompletionException: io.atomix.cluster.messaging.MessagingException$NoRemoteHandler: No remote message handler registered for this message 
I 2021-09-16T14:47:23.977986233Z {} 
I 2021-09-16T14:47:26.421742Z RaftServer{raft-partition-partition-1}{role=FOLLOWER} - No heartbeat from 1 since 3861ms 
I 2021-09-16T14:47:26.422383Z RaftServer{raft-partition-partition-1}{role=FOLLOWER} - Sending poll requests to all active members: [DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}, DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}] 
I 2021-09-16T14:47:26.429009Z RaftServer{raft-partition-partition-1} - Transitioning to CANDIDATE 
I 2021-09-16T14:47:26.429895Z RaftServer{raft-partition-partition-1}{role=CANDIDATE} - Starting election 
I 2021-09-16T14:47:26.449386Z RaftServer{raft-partition-partition-1} - Transitioning to LEADER 
I 2021-09-16T14:47:26.450141Z RaftServer{raft-partition-partition-1} - Found leader 2 
I 2021-09-16T14:47:26.504880Z Close appender for log stream logstream-raft-partition-partition-1 
I 2021-09-16T14:47:26.505428Z On closing logstream logstream-raft-partition-partition-1 close 1 readers 
I 2021-09-16T14:47:26.685125Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@5c433843)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@f80e2b1, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@548f33f7, configuration: Configuration(false)] 
I 2021-09-16T14:47:26.687063Z Recovered state of partition 1 from snapshot at position 73462 
I 2021-09-16T14:47:26.687648Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@76d6b0f2)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@33921f4b, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@3b0ff6e2, configuration: Configuration(false)] 
I 2021-09-16T14:47:26.688783Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@16327aff)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@55a019eb, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@76eb3c53, configuration: Configuration(false)] 
I 2021-09-16T14:47:26.689934Z Engine created. [value-mapper: CompositeValueMapper(List(io.camunda.zeebe.el.impl.feel.MessagePackValueMapper@32976c8a)), function-provider: io.camunda.zeebe.el.impl.feel.FeelFunctionProvider@37771da5, clock: io.camunda.zeebe.el.impl.ZeebeFeelEngineClock@624d25e1, configuration: Configuration(false)] 
I 2021-09-16T14:47:26.692583Z Processor starts replay of events. [snapshot-position: 73462, replay-mode: PROCESSING] 
I 2021-09-16T14:47:26.693311Z Processor finished replay at event position 73464 
I 2021-09-16T14:47:26.693596Z Starting processing of migration tasks (use LogLevel.DEBUG for more details) ...  
I 2021-09-16T14:47:26.695426174Z {} 
I 2021-09-16T14:47:26.695432Z Skipping MessageSubscriptionSentTimeMigration migration (2/2).  It was determined it does not need to run right now. 
I 2021-09-16T14:47:26.695694Z Completed processing of migration tasks (use LogLevel.DEBUG for more details) ...  
I 2021-09-16T14:47:26.698906Z Exporter opened 
W 2021-09-16T14:47:27.437934Z RaftServer{raft-partition-partition-1} - ConfigureRequest{term=6, leader=2, index=0, timestamp=1631623859958, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}]} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-1.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:47:27.438727Z RaftServer{raft-partition-partition-1} - AppendRequest{term=6, leader=2, prevLogIndex=25180, prevLogTerm=5, entries=0, commitIndex=25180} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-1.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:47:28.452704Z RaftServer{raft-partition-partition-1} - ConfigureRequest{term=6, leader=2, index=0, timestamp=1631623859958, members=[DefaultRaftMember{id=1, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}, DefaultRaftMember{id=2, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}, DefaultRaftMember{id=0, type=ACTIVE, updated=2021-09-14T12:50:59.958Z}]} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Failed to connect channel for address zeebe-1.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502 
W 2021-09-16T14:47:41.212453Z request [PUT http://elasticsearch:9200/_component_template/zeebe-record] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.254202Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_process_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.259089Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_error_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.263922Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_incident_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.268699Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_job_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.273046Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_variable_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.277887Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_variable-document_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.282219Z request [PUT http://elasticsearch:9200/_index_template/zeebe-record_process-instance_1.2.0-alpha2] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:41.774583Z request [POST http://elasticsearch:9200/_bulk] returned 1 warnings: [299 Elasticsearch-7.13.2-4d960a0733be83dd2543ca018aa4ddc42e956800 "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.13/security-minimal-setup.html to enable security."] 
W 2021-09-16T14:47:42.987642Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13007, prevLogTerm=10, entries=0, commitIndex=13007} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Expected to send a message with subject 'raft-partition-partition-2-append' to member '1', but member is not known. Known members are '[Member{id=2, address=zeebe-2.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwACAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMi56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAAIAAAAADAACAQAAAAYAAAAAAAAAAgAAAAoAAAAAAAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}, Member{id=0, address=zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwAAAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMC56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAQIAAAABDAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}]'. 
W 2021-09-16T14:47:53.702185Z RaftServer{raft-partition-partition-1} - AppendRequest{term=6, leader=2, prevLogIndex=25180, prevLogTerm=5, entries=0, commitIndex=25187} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Expected to send a message with subject 'raft-partition-partition-1-append' to member '1', but member is not known. Known members are '[Member{id=2, address=zeebe-2.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwACAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMi56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAAIAAAAADAACAQAAAAYAAAAAAAAAAgAAAAoAAAAAAAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}, Member{id=0, address=zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwAAAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMC56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAQIAAAABDAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}]'. 
W 2021-09-16T14:47:55.487626Z RaftServer{raft-partition-partition-2} - AppendRequest{term=10, leader=2, prevLogIndex=13007, prevLogTerm=10, entries=0, commitIndex=13007} to 1 failed: java.util.concurrent.CompletionException: java.net.ConnectException: Expected to send a message with subject 'raft-partition-partition-2-append' to member '1', but member is not known. Known members are '[Member{id=2, address=zeebe-2.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwACAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMi56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAAIAAAAADAACAQAAAAYAAAAAAAAAAgAAAAoAAAAAAAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}, Member{id=0, address=zeebe-0.zeebe-broker-service.3300ffcb-1c2f-4ccd-9e57-d3bc532ecc86-zeebe.svc.cluster.local:26502, properties={brokerInfo=EADJAAAAAwAAAAAAAgAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGlfAAAAemVlYmUtMC56ZWViZS1icm9rZXItc2VydmljZS4zMzAwZmZjYi0xYzJmLTRjY2QtOWU1Ny1kM2JjNTMyZWNjODYtemVlYmUuc3ZjLmNsdXN0ZXIubG9jYWw6MjY1MDEFAAIBAAAAAQIAAAABDAAADAAAADEuMi4wLWFscGhhMgUAAgEAAAABAgAAAAE=, event-service-topics-subscribed=KIIDAGpvYnNBdmFpbGFibOU=}}]'. 
I 2021-09-16T14:48:00.230090783Z {} 
I 2021-09-16T14:48:26.750323Z Finished taking temporary snapshot, need to wait until last written event position 73483 is committed, current commit position is 73483. After that snapshot will be committed. 
I 2021-09-16T14:48:26.772506Z Committed new snapshot 25185-6-73480-73483 
E 2021-09-16T14:49:41.124382Z Expected to successfully process event 'LoggedEvent [type=0, version=16384, streamId=1, position=73491, key=2251799813712467, timestamp=1631803781109, sourceEventPosition=73487] RecordMetadata{recordType=COMMAND, intentValue=255, intent=ACTIVATE_ELEMENT, requestStreamId=-2147483648, requestId=-1, protocolVersion=3, valueType=PROCESS_INSTANCE, rejectionType=NULL_VAL, rejectionReason=, brokerVersion=1.2.0}' with processor, but caught an exception. Skip this event. 
E 2021-09-16T14:49:41.151216Z Expected to process event 'TypedEventImpl{metadata=RecordMetadata{recordType=COMMAND, intentValue=255, intent=ACTIVATE_ELEMENT, requestStreamId=-2147483648, requestId=-1, protocolVersion=3, valueType=PROCESS_INSTANCE, rejectionType=NULL_VAL, rejectionReason=, brokerVersion=1.2.0}, value={"bpmnProcessId":"timerProcess","version":1,"processDefinitionKey":2251799813686212,"processInstanceKey":2251799813712467,"elementId":"EndEvent_02qhg5x","flowScopeKey":2251799813712467,"bpmnElementType":"END_EVENT","parentProcessInstanceKey":-1,"parentElementInstanceKey":-1}}' without errors, but exception occurred with message 'The Processor for the element type END_EVENT is already migrated no need to call #transitionToActivating again this is already done in the BpmnStreamProcessor for you. Happy to help :) '. 
W 2021-09-16T14:49:41.152896Z Blacklist process instance 2251799813712467, due to previous errors. 

```

**Comment by npepinpe** [comment]
Assumption is that if our theory was right, then it should have been fixed since then. Let's close it, and if it pops again in 1.2.0 we can reopen this.

**Comment by deepthidevaki** [comment]
Found this in release candidate benchmark https://console.cloud.google.com/errors/CKCR7Y-Xzqq6Eg?service=zeebe-broker&version=release-1-2-0&time=P7D&project=zeebe-io

**Comment by npepinpe** [comment]
@saig0 as :ambulance: can you look into it? It seems this wasn't fixed. Do we have any new ideas on the likelihood? It doesn't seem like a release blocker from my POV, but I have to admit it's unclear what the likelihood here is.

**Comment by saig0** [comment]
It occurred again in the medic benchmark CW40: https://console.cloud.google.com/errors/CKCR7Y-Xzqq6Eg?service=zeebe-broker&version=medic-cw-40-ab56d66c4b-benchmark&time=P7D&project=zeebe-io

**Comment by korthout** [comment]
Marking priority as `later later` because it is blocked by #8005 which also has priority `later later`.

Please comment if you think this should have a higher priority.
